rosbag2_storage_default_plugins/src/rosbag2_storage_default_plugins/sqlite/sqlite_storage.cpp

```diff
@@ -101,6 +106,34 @@ void SqliteStorage::open(
     relative_path_ << "' for " << to_string(io_flag) << ".");
 }
 
+void SqliteStorage::activate_transaction()
+{
+  int rc = -1;
```

Comment:
This line can be shifted within the `if` block, right?

Reply:
Also for clarity, I would prefer having it initialized with `SQLITE_ERROR`. https://www.sqlite.org/rescode.html#error